src/app/zap-templates/gen_config.zapt

```diff
@@ -16,3 +16,48 @@
 {{#all_user_clusters}}
 #define EMBER_AF_{{asDelimitedMacro define}}_{{asDelimitedMacro side}}_ENDPOINT_COUNT ({{user_endpoint_count_by_cluster id side}})
 {{/all_user_clusters}}
+
+/**** Cluster Plugins ****/
+{{#all_user_clusters}}
+
+// Use this macro to check if {{side}} plugin {{name}} is included
```

Comment:
Maybe "to check if the {{side}} side of the {{name}} cluster is included"?  At some point we're going to want to change the "plugin" naming to something else, possibly....

Reply:
> Maybe "to check if the {{side}} side of the {{name}} cluster is included"? At some point we're going to want to change the "plugin" naming to something else, possibly....

Done.

Also I have not bothered changing the name yet but I agree with you. Plugin does not makes sense in our context.